packages/gimbal/src/command/index.ts

```diff
@@ -30,6 +35,43 @@ interface Config {
   title: string;
 }
 
+const reconcileReports = (reports: Report | Report[]): Report => {
+  if (Array.isArray(reports)) {
+    const [first, ...rest] = reports;
+
+    rest.forEach(
+      (report: Report): void => {
+        if (!report.success) {
+          first.success = report.success;
+        }
+
+        if (Array.isArray(report.data) && report.data.length) {
+          report.data.forEach(
+            (reportItem: ReportItem): void => {
+              const match =
+                first.data && first.data.find((firstItem: ReportItem): boolean => firstItem.type === reportItem.type);
+
+              if (match) {
+                reconcile(match, reportItem, reports.length);
```

Comment:
Instead of `reports.length` for a total number, we need to pass a dynamic number (maybe `index + 1` for the current report plus the first report).